akka-actor-typed-tests/src/test/scala/akka/actor/typed/internal/ActorSystemSpec.scala

```diff
@@ -69,21 +71,35 @@ class ActorSystemSpec extends WordSpec with Matchers with BeforeAndAfterAll with
 
     "terminate the guardian actor" in {
       val inbox = TestInbox[String]("terminate")
-      val sys = system(
-        Behaviors
-          .receive[Probe] {
-            case (_, _) => Behaviors.unhandled
-          }
-          .receiveSignal {
-            case (_, PostStop) =>
-              inbox.ref ! "done"
-              Behaviors.same
-          },
-        "terminate")
+      val sys = system(Behaviors.setup[Any] { _ =>
+        inbox.ref ! "started"
```

Comment:
changed the failing test, so that we first know that it has been started before we terminate the ActorSystem